aw-server/tests/api.rs

```diff
@@ -239,6 +253,7 @@ mod api_tests {
         res = client
             .get("/api/0/buckets/id/events")
             .header(ContentType::JSON)
+            .header(Header::new("Host", "127.0.0.1:5600"))
```

Comment:
You needed to add this explicitly? The `Host` header should be set correctly automatically, and you shouldn't be testing against port 5600, right?

Reply:
Tried removing them, which caused the tests to break.

Reply:
>  and you shouldn't be testing against port 5600, right?

The port or address actually doesn't matter. We are never starting an actual server or starting any ports in these unit tests, they are simply injected into Rocket as if they were real requests.

> The Host header should be set correctly automatically

Since the testing framework is a very basic I'll assume that they either simply have not seen this issue before (since the requests are internal to rocket, the Host header is technically pointless) or simply because the Rocket developers find it to make more sense for all headers to be explicit rather than implicit.

Reply:
Ah, I suspected something like that, but found it a bit weird and couldn't quite explain why it would work that way when the `Host` header is so well specified and ubiquitous.

>or simply because the Rocket developers find it to make more sense for all headers to be explicit rather than implicit.

This would seem odd to me in any other context where you make requests, but perhaps it makes sense for tests like these.